composer.json

```diff
@@ -13,14 +13,13 @@
     "keywords": ["hammock", "hack", "hacklang", "library", "mock", "mocking", "test", "testing", "php"],
     "type": "library",
     "require": {
-        "hhvm/hsl": "^3.30",
-        "hhvm/hsl-experimental": "^3.29",
-        "hhvm/hhvm-autoload": "^1.8",
-        "facebook/hh-clilib": "^1.2.1|^2"
+		  	"hhvm": ">=3.24",
+		  	"hhvm/hhvm-autoload": "^1|^2|^3",
+		  	"hhvm/hsl": ">1.4 <5"
```

Comment:
And consistent indentation here?

Reply:
Yeah, this is a classic case of _it looks good with my tab width._
I switch tab widths back and forth all the time, since 2 is the Hack default,
but 4 is what we use for our stuff.
This is something I overlooked.
Thanks for telling me about how bad these files looked.